tests/test-location.js

```diff
@@ -26,12 +26,12 @@ tape.test('testing new view mapping, convenience functions, and member functions
     t.plan(2)
     q.series([
     function(lib) {
-        server.listen(8082,function(){
+        server.listen(8084,function(){
```

Comment:
Is there a reason this needed to change? My guess is that you changed this locally to get a test working after a prior one had been left hanging and it ended up in the pull request :)